src/System.Numerics.Matrices/src/System/Numerics/IMatrix.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace System.Numerics.Matrices
+{
+    interface IMatrix
+    {
+        double this[int col, int row] { get; set; }
```

Comment:
Anyway, I will merge the PR. The goal of corefx lab projects IMO is to start with what's possible to day, force discussions on limitations, and then possibly try to fix solutions for these limitations.